### PR TITLE
Rov 191 create a calculator to calculate monthly emi

### PR DESCRIPTION
**Pull request by Varuno8**

### CODE REVIEW

File: src/app/application/emi-calculator/emi-calculator.component.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<h1 class="emi text-bold">EMI Calculator</h1>
+<p class="page_subtitle  text-subtitle">Calculate monthly EMI</p>
```
Comment (Vedant-Tyagi) [inline src/app/application/emi-calculator/emi-calculator.component.html:2]: change the font weight it should be more bold

File: src/app/application/emi-calculator/emi-calculator.component.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<h1 class="emi text-bold">EMI Calculator</h1>
```
Comment (Vedant-Tyagi) [inline src/app/application/emi-calculator/emi-calculator.component.html:1]: page should be responsive

**Comment by yaneshtyagi** [comment]
@Vedant-Tyagi please review this PR.

**Comment by Vedant-Tyagi** [comment]
@yaneshtyagi  this card is still in process may be varun is  working on this card once he completes his work and pushed it into review then I will be reviewing this

**Review by Vedant-Tyagi — DISMISSED** [review]
It seems to be working fine and I found no error

**Comment by yaneshtyagi** [comment]
@Vedant-Tyagi What is the responsibility of `CalculatorComponent` class? Is the name of the class appropriate?

**Comment by yaneshtyagi** [comment]
> @Vedant-Tyagi What is the responsibility of `CalculatorComponent` class? Is the name of the class appropriate?

@Vedant-Tyagi Please respond to this comment.

**Review by yaneshtyagi — CHANGES_REQUESTED** [review]
Please resolve the conflicts.

**Comment by Vedant-Tyagi** [comment]
> > @Vedant-Tyagi What is the responsibility of `CalculatorComponent` class? Is the name of the class appropriate?
> 
> @Vedant-Tyagi Please respond to this comment.
> @yaneshtyagi  This calculator is used to determine the Monthly EMI, the Total Interest Paid, and the Total Amount, which is the product of the principal amount and the total time period.

**Comment by yaneshtyagi** [comment]
When you are resolving the merge conflicts, you should have understood that naming a class appropriately is important. Generic names like `CalculatorComponent` cause the problems since we have many calculators. I expect such issues to be pointed out during code review.

**Comment by Varuno8** [comment]
> When you are resolving the merge conflicts, you should have understood that naming a class appropriately is important. Generic names like `CalculatorComponent` cause the problems since we have many calculators. I expect such issues to be pointed out during code review.

it was my first card, apologies for that mistake, I pointed the same mistake on the next card

**Comment by yaneshtyagi** [comment]
No need for apologies. It's all part of learning.

**Comment by Vedant-Tyagi** [comment]
@yaneshtyagi  I have reviewed and the conflict has been resolved and functionality is also working fine

**Review by Vedant-Tyagi — APPROVED** [review]

**Review by Vedant-Tyagi — APPROVED** [review]

**Review by yaneshtyagi — APPROVED** [review]

**Pull request merged**